.happy/terraform/modules/sfn_config/gisaid.wdl

```diff
@@ -181,21 +183,23 @@ task TransformGISAID {
     mv /ncov-ingest/source-data/czgenepi_locations.tsv \
        /ncov-ingest/source-data/gisaid_geoLocationRules.tsv
 
-    # decompress the gisaid dataset and transform it.
-    ${aws} s3 cp --no-progress "s3://${raw_gisaid_s3_bucket}/${raw_gisaid_s3_key}" - | zstdmt -d > gisaid.ndjson
-    /ncov-ingest/bin/transform-gisaid     \
-        gisaid.ndjson                     \
+    # decompress the raw dataset and transform it.
+    ${aws} s3 cp --no-progress "s3://${raw_data_s3_bucket}/${raw_data_s3_key}" - | zstdmt -d > genbank.ndjson
```

Comment:
Nit (that I might even retract if I had more context): everything else has moved pretty carefully to really generalized language of "raw dataset" etc instead of saying where the data came from, but this switches back to saying genbank explicitly. That said, I expect this `transform-genbank` call is inherently built to working against genbank, not generalized, so trying to name things generically here is probably just silly. But maybe a comment saying we're specifically built with genbank in mind for the time being would make sense somewhere? I dunno.